### ISSUE TITLE
[BUG] Not working anymore with development-v6 branch of pihole

### ISSUE THREAD

**Issue by coffeinflash** [tien/pi-hole-connect#179]
**Describe the bug**
with the v6-development branch of pihole(https://github.com/pi-hole/AdminLTE/tree/development-v6) the complete Admin-LTE changed. Due to this the login does not work anymore.

I guess the API-endpoint is wrong now and the queries to get the overview also.

Not sure if this is related: https://pi-hole.net/blog/2022/11/17/upcoming-changes-authentication-for-more-api-endpoints-required/

**Comment by tien** [comment]
Yep since V6 is in beta now, I'll spend some time to get this to work
